src/druidRequester.ts

```diff
@@ -432,7 +432,7 @@ export function druidRequesterFactory(parameters: DruidRequesterParameters): Ply
                           if (body && typeof body.host === 'string') error.host = body.host;
                         }
                       } catch (e) {
-                        error = new Error(`bad response: ${e.message}`);
+                        error = new Error(`bad response: ${resp}`);
```

Comment:
Instead of replacing the error message completely, could we augment it with the response? Or will these errors always have the same message?

```suggestion
                        error = new Error(`bad response: ${e.message}\n${resp}`);
```